.github/workflows/managed.yml

```diff
@@ -1,9 +1,5 @@
 name: Managed
 on:
-  schedule:
-    # run every Sunday to re-populate caches after they are cleaned on Saturday
-    - cron: "0 12 * * 0"
-
```

Comment:
The schedule is no longer relevant, since the cleanup action was removed earlier.